doc/src/modules/polys/basics.rst

```diff
@@ -14,6 +14,319 @@ polynomials within SymPy. All code examples assume::
     >>> x, y, z = symbols('x,y,z')
     >>> init_printing(use_unicode=False, wrap_line=False, no_global=True)
 
+Basic concepts
+==============
+
+Polynomials
+-----------
+
+Given a family `(x_i)` of symbols, or other suitable objects,
+expressions derived from them by repeated addition, subtraction
+and multiplication are called *polynomial expressions in the
+generators* `x_i`.
```

Comment:
I tried to make a distinction between 'polynomial expressions' and 'polynomials'. The belong to different classes in SymPy. Adding 'including numbers' here seems a good idea. But I did not want to introduce base rings at this point.
(Writing expository texts is always balancing between brevity and completeness.)